### PR TITLE
OP-1165-fix-circular-reference-in-jpa-config

### PR DESCRIPTION
**Pull request by mwithi**
Just for testing OP-1165.

Possibly, merging https://github.com/informatici/openhospital-api/pull/422/commits/d29a394ab0610aa3a23e56ff45573f6246af124f

### CODE REVIEW

File: .github/workflows/maven.yml
Comment (dbmalkovsky) [inline .github/workflows/maven.yml:18]: doesn't this need to be changed before merge?

**Review by dbmalkovsky — APPROVED** [review]
LGTM after comment is addressed

File: .github/workflows/maven.yml
Comment (mwithi) [reply .github/workflows/maven.yml:18]: yes, of course, if the change to `src/main/java/org/isf/config/SecurityConfig.java` is also neeed, otherwise we can just close this PR becuase the problem was actually in core only.

**Comment by mwithi** [comment]
Closing because not needed.

> Just for testing [OP-1165](https://openhospital.atlassian.net/browse/OP-1165).
> 
> Possibly, merging [d29a394](https://github.com/informatici/openhospital-api/commit/d29a394ab0610aa3a23e56ff45573f6246af124f)

Closing because tests are ok and change in d29a394](https://github.com/informatici/openhospital-api/commit/d29a394ab0610aa3a23e56ff45573f6246af124f is not really needed.

**Pull request closed without merging**
